tap_pendo/streams.py

```diff
@@ -219,7 +220,7 @@ def __init__(self, config=None):
         self.config = config
 
     def send_request_get_results(self, req):
-        resp = session.send(req)
+        resp = session.send(req, timeout=300)
```

Comment:
In case of this, we need to handle the timeout exception.

Reply:
Backoff for Timeout error is already included in requests.exceptions.RequestException as it is the parent class of 'Timeout' error.